packages/contracts-bedrock/src/L2/L2StandardBridge.sol

```diff
@@ -52,20 +52,24 @@ contract L2StandardBridge is StandardBridge, ISemver {
         bytes extraData
     );
 
-    /// @custom:semver 1.7.0
-    string public constant version = "1.7.0";
+    /// @custom:semver 1.8.0
+    string public constant version = "1.8.0";
 
     /// @notice Constructs the L2StandardBridge contract.
     /// @param _otherBridge Address of the L1StandardBridge.
-    constructor(address payable _otherBridge)
-        StandardBridge(payable(Predeploys.L2_CROSS_DOMAIN_MESSENGER), _otherBridge)
-    {
-        initialize();
+    constructor(address payable _otherBridge) StandardBridge() {
+        initialize({
+            _messenger: CrossDomainMessenger(payable(Predeploys.L2_CROSS_DOMAIN_MESSENGER)),
```

Comment:
This should be `address(0)` in the singleton

Reply:
should we drop the parameter to the constructor or just set `_messenger` pointing to `address(0)`?